### PR TITLE
Pr/behat test

### PR DESCRIPTION
**Pull request by eliatib**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -25,6 +25,8 @@ matrix:
               - SYLIUS_VERSION="1.4.*"
         - env:
               - SYLIUS_VERSION="1.5.*"
+        - env:
+              - SYLIUS_VERSION="1.6.*"
```
Comment (ondrej-kuhnel) [inline .travis.yml:30]: uncomment `- bin/behat`

**Review by ondrej-kuhnel — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (eliatib) [reply .travis.yml:30]: DONE

File: .idea/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```
Comment (ondrej-kuhnel) [inline .idea/.gitignore]: remove `.idea` folder

File: behat.yml.dist [hunk 3]
```diff
@@ -1,21 +1,47 @@
...
+        Behat\MinkExtension:
+            files_path: "%paths.base%/vendor/sylius/sylius/src/Sylius/Behat/Resources/fixtures/"
+            base_url: "http://localhost:8080/"
+            default_session: symfony
+            javascript_session: chrome
+            sessions:
+                symfony:
+                    symfony: ~
+                chrome:
+                    selenium2:
```
Comment (ondrej-kuhnel) [inline behat.yml.dist]: I don't think you're using selenium?

File: features/managing_sorting/disable_product.feature [hunk 4]
```diff
@@ -0,0 +1,19 @@
+@sorting
```
Comment (ondrej-kuhnel) [inline features/managing_sorting/disable_product.feature]: Join both feature files

File: features/managing_sorting/disable_product.feature [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+	  Given the store operates on a single channel in "United States"
+	  And the store has "mens_jeans" taxonomy
+	  And the store has a "Slim fit classic" configurable product
+	  And this product is in "mens_jeans" taxon at 1st position
+	  And this product has been disabled
+	  And I am logged in as an administrator
+
+  @ui
+  Scenario: Being able to sort products
+    When I open the "mens_jeans" taxon page
```
Comment (ondrej-kuhnel) [inline features/managing_sorting/disable_product.feature]: taxon sorting page?

File: features/managing_sorting/disable_product.feature [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+  Background:
+	  Given the store operates on a single channel in "United States"
+	  And the store has "mens_jeans" taxonomy
+	  And the store has a "Slim fit classic" configurable product
+	  And this product is in "mens_jeans" taxon at 1st position
+	  And this product has been disabled
+	  And I am logged in as an administrator
+
+  @ui
+  Scenario: Being able to sort products
```
Comment (ondrej-kuhnel) [inline features/managing_sorting/disable_product.feature]: You mean see disabled products?

File: features/managing_sorting/sort_products.feature [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+	  And the store has "mens_jeans" taxonomy
+	  And the store has a "Slim fit classic" configurable product
+	  And this product is in "mens_jeans" taxon at 1st position
+	  And the store has a "Slim fit elegant" configurable product
+	  And this product is in "mens_jeans" taxon at 2nd position
+	  And I am logged in as an administrator
+
+  @ui
+  Scenario: Being able to sort products
+    When I open the "mens_jeans" taxon page
```
Comment (ondrej-kuhnel) [inline features/managing_sorting/sort_products.feature]: It should start with GIVEN

File: features/managing_sorting/disable_product.feature (hunk 5 above)
Comment (ondrej-kuhnel) [inline features/managing_sorting/disable_product.feature]: It should start with GIVEN

File: tests/Behat/Pages/Admin/Sorting/SortingPageInterface.php [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+
+declare(strict_types=1);
+
+namespace Tests\MangoSylius\SortingPlugin\Behat\Pages\Admin\Sorting;
+
+use FriendsOfBehat\PageObjectExtension\Page\SymfonyPageInterface;
+
+interface SortingPageInterface extends SymfonyPageInterface
+{
+	public function saveSorting();
```
Comment (ondrej-kuhnel) [inline tests/Behat/Pages/Admin/Sorting/SortingPageInterface.php]: missing return types

File: features/managing_sorting/disable_product.feature [hunk 9]
```diff
@@ -0,0 +1,19 @@
+@sorting
+Feature: Sort products note
```
Comment (ondrej-kuhnel) [inline features/managing_sorting/disable_product.feature]: note?

File: features/managing_sorting/disable_product.feature [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+  In order to sort products in the store
```
Comment (ondrej-kuhnel) [inline features/managing_sorting/disable_product.feature]: in category

File: features/managing_sorting/sort_products.feature (hunk 7 above)
Comment (ondrej-kuhnel) [reply features/managing_sorting/sort_products.feature]: Add after this `And I should see the "Slim fit classic" in 1st position` (yes duplicate :))

**Review by ondrej-kuhnel — CHANGES_REQUESTED** [review]

File: .idea/.gitignore (hunk 2 above)
Comment (eliatib) [reply .idea/.gitignore]: DONE

File: behat.yml.dist (hunk 3 above)
Comment (eliatib) [reply behat.yml.dist]: it's from the standard behat.yml.dist that I used, I removed all selenium parts

File: features/managing_sorting/disable_product.feature (hunk 4 above)
Comment (eliatib) [reply features/managing_sorting/disable_product.feature]: DONE

File: features/managing_sorting/disable_product.feature (hunk 5 above)
Comment (eliatib) [reply features/managing_sorting/disable_product.feature]: yes, I changed it

File: features/managing_sorting/disable_product.feature (hunk 6 above)
Comment (eliatib) [reply features/managing_sorting/disable_product.feature]: yes, I changed it

File: features/managing_sorting/sort_products.feature (hunk 7 above)
Comment (eliatib) [reply features/managing_sorting/sort_products.feature]: DONE

File: tests/Behat/Pages/Admin/Sorting/SortingPageInterface.php (hunk 8 above)
Comment (eliatib) [reply tests/Behat/Pages/Admin/Sorting/SortingPageInterface.php]: DONE

File: features/managing_sorting/disable_product.feature (hunk 10 above)
Comment (eliatib) [reply features/managing_sorting/disable_product.feature]: DONE

File: features/managing_sorting/disable_product.feature (hunk 9 above)
Comment (eliatib) [reply features/managing_sorting/disable_product.feature]: it's an oversight, I removed it,

File: features/managing_sorting/disable_product.feature (hunk 5 above)
Comment (eliatib) [reply features/managing_sorting/disable_product.feature]: DONE

File: features/managing_sorting/sort_products_and_show_disable_products.feature [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+	  And the store has a "Slim fit classic" configurable product
+	  And this product is in "mens_jeans" taxon at 1st position
+	  And this product has been disabled
+	  And the store has a "Slim fit elegant" configurable product
+	  And this product is in "mens_jeans" taxon at 2nd position
+	  And I am logged in as an administrator
+
+	@ui
+	Scenario: Being able to sort products
+	Given I open the "mens_jeans" taxon sorting page
```
Comment (ondrej-kuhnel) [inline features/managing_sorting/sort_products_and_show_disable_products.feature]: add spaces (format it)

**Review by ondrej-kuhnel — CHANGES_REQUESTED** [review]

File: features/managing_sorting/sort_products_and_show_disable_products.feature (hunk 11 above)
Comment (eliatib) [reply features/managing_sorting/sort_products_and_show_disable_products.feature]: DONE

**Review by ondrej-kuhnel — APPROVED** [review]

**Pull request merged**